### DISCUSSION TITLE
How can I execute Neovim using the Command() function?

### DISCUSSION THREAD

**Discussion by Jaehaks** [sxyazi/yazi#1715] [1. Q&A] [answered]
Hi :)

I try to execute subprocess like neovim in yazi.
I make the plugin like this 

```lua
return {
  entry = function()
    local paths = selected_url()
    if not paths then
      info('Diff Error : select two files')
      return
    end
    
    local _permit = ya.hide()
    local status, err = Command("nvim"):arg("-d")
                        :arg(paths[1]):arg(paths[2])
                        :spawn():wait()
    -- if error then
    -- 	return info("Failed to run diff, error: " .. err)
    -- end
    _permit:drop()
  end,
}

```
After executing this plugin,
Yazi is hided and neovim process executes as I think. because `nvim.exe` is added in process list. 
But the screen has forzen after yazi is hided while showing only cmd prompt line. I am windows user. 

like this
![image](https://github.com/user-attachments/assets/def4f18a-03bc-4a7c-afb4-77669c34b68b)

After I force to terminate the nvim.exe process, the yazi screen is restored
What did I miss it?

**Comment by sxyazi** [comment, marked as answer]
Neovim is an interactive app, so you'll need to allow it to inherit your tty resource, try adding:
`:stdin(Command.INHERIT):stdout(Command.INHERIT):stderr(Command.INHERIT)` to your `Command`

**Reply by Jaehaks** [reply]
Perfect solution:)
